### PR TITLE
Simplify option handling

### PR DESCRIPTION
**Pull request by gforcada**
I'm having a look at the code and I saw a pattern repeated, so I though it can be simplified a little bit, to make it a bit more readable 🍀

### CODE REVIEW

**Review by icemac — APPROVED** [review]
I like the idea of simplification but added a commit to the PR so the function does not rely on global state and gets a docstring.

File: config/config-package.py [hunk 1]
```diff
@@ -190,26 +190,30 @@ def copy_with_meta(
...
 
+
+def set_python_config_value(meta_cfg, args, name, default=False):
+    """Get value from either python section in config file or cmd line arg.
+
+    Return the value and store it in `meta_cfg`.
+    """
+    key = f'with-{name}'
+    existing_value = meta_cfg['python'].get(key, False)
+    arg_value = vars(args)[f"with_{name.replace('-', '_')}"]
```
Comment (mgedmin) [inline config/config-package.py:201]:
```suggestion
    arg_value = getattr(args, key.replace('-', '_'))
```

**Review by mgedmin — APPROVED** [review]
LGTM, but I also suggest a simplification for a single statement that makes it more readable for me.

**Comment by gforcada** [comment]
Thanks for the review and improvements suggested/applied ✨

**Pull request merged**